src/components/Login/index.tsx

```diff
@@ -1,31 +1,30 @@
-import { USERS } from 'fakeData/users';
+import userApi from 'api/userApi';
+import { useAppDispatch } from 'app/hooks';
+import { getUserInfo } from 'features/user/userSlice';
 import React from 'react';
 import { useHistory } from 'react-router-dom';
-import { login } from 'utils/auth';
 import './Login.scss';
 import { useLoginForm } from './useLoginForm';
-
 function Login() {
    const history = useHistory();
+   const dispatch = useAppDispatch();
```

Comment:
```suggestion
   const { login, loading } = useLogin();
```

Khi loading = true thì hiển thị overlay loading trên màn hình nhé
![image](https://user-images.githubusercontent.com/31428578/138203027-bc703cd2-67b3-4c53-a187-51df0d3415c8.png)